inc/features/dcbreadcrumb.class.php

```diff
@@ -151,6 +154,52 @@ public function isEnclosurePart($itemtype, $items_id, $getobj = false) {
       return $found;
    }
 
+   /**
+    * get item position from Enclosure
+    *
+    * @param string  $itemtype Item type
+    * @param integer $items_id Item ID
+    *
+    * @return string
+    */
+   public function getItemEnclosurePosition($itemtype, $items_id) {
```

Comment:
I guess those two new methods should be unit tested

Reply:
adding TU for breadcrumb